package.json

```diff
@@ -5,6 +5,9 @@
     "tag": "lts"
   },
   "description": "LoopBack Connector for IBM Cloundant DB",
+  "engines": {
+    "node": ">= 4 <= 6"
```

Comment:
Any reason this doesn't work with 8?

Reply:
@virkt25 This is branch 1.x and I copied the engines from `loopback-datasource-juggler@2.x`, I think they have the same engines support

Reply:
`1.x` can still be used with `3.x` version of Juggler. I disagree with it only supporting what 2.x supports because 3.x does support Node 8. Unless this connector version of 1.x shouldn't be used with Juggler 3.x at all.

Reply:
The code of `1.x` is written when node@8.x was not released, and due to the annoying bug of `automigrate`, I would not recommend new users to use it, `1.x` doesn't have the index and view feature either. That's why I don't want to spend effort on its compatibility of node 8, even it passes all test on node 8, I still don't want to give loopback3 users an impression that this old version is an option for them :(